modules/services/polybar.nix

```
@@ -205,7 +205,8 @@ in {
 
       Service = {
         Type = "forking";
-        Environment = "PATH=${cfg.package}/bin:/run/wrappers/bin";
+        Environment =
+          "PATH=\${PATH:+:}${cfg.package}/bin:/run/wrappers/bin NIX_PATH=\${NIX_PATH:+:}";
```

```suggestion
          "PATH=$PATH\${PATH:+:}${cfg.package}/bin:/run/wrappers/bin";
```
The `\${PATH:+:}` substitutes the value of the environment variable `PATH` if it exists and is set to `:`. This is why `$PATH\${PATH:+:}` is required.

`NIX_PATH` is inherited from the systemd user's session. So the `NIX_PATH` logic is a noop.